File: ConcessionariaAPI/ConcessionariaAPI/Models/Locacao.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace ConcessionariaAPI.Models
{
    public partial class Locacao
    {
        public int Locacaoid { get; set; }
        public int? Clienteid { get; set; }
        public int? Veiculoid { get; set; }
        public string Observacao { get; set; }
        public decimal? Valorlocacao { get; set; }
        public DateTime? DtLocacao { get; set; }
        public DateTime? DtDevolucao { get; set; }
        public DateTime? DtRecord { get; set; }
        public DateTime? DtUpdate { get; set; }

        [JsonIgnore]
        public Cliente Cliente { get; set; }
        [JsonIgnore]
        public Veiculo Veiculo { get; set; }
    }
}

File: WebApp/src/app/cadastros/locacao/model/locacao.ts
import { Cliente } from "../../cliente/model/cliente";
import { Veiculo } from "../../veiculo/model/veiculo";

export class Locacao  {
    locacaoid: number;
    cliente: Cliente;
    veiculo: Veiculo;
    observacao: string;
    valorlocacao: string;
    dtLocacao: Date;
    dtDevolucao: Date;
}
